docs/freqai-configuration.md

```diff
@@ -0,0 +1,283 @@
+# Configuration
+
+## Setting up the configuration file
+
+The user interface is isolated to the typical Freqtrade config file. Although there are plenty of additional parameters that a user can choose from, as highlighted in the [parameter table](#parameter-table), a FreqAI config must at minimum include the following parameters (the example inputs are not requires):
```

Comment:
> The user interface is isolated to the typical Freqtrade config file

that "isolated" seems missplaced / misworded - I've read the whole paragraph 3 times now - and i still not sure in what way it's isolated. it's a separate section - sure, but ...

(it's also a lie, the "user interface" is not just the config file - but also the strategy - which takes part in freqAI's success/failure just as much).

Reply:
@th0rntwig fixed this